java/arcs/core/allocator/Arc.kt

```diff
@@ -53,19 +56,18 @@ class Arc internal constructor(
      *  is considered to be [Error].
      */
     var arcState: ArcState
-        get() = arcStateInternal
-        private set(state) = sync(this) {
-            if (arcStateInternal != state) {
-                arcStateInternal = state
-                fireArcStateChange()
```

Comment:
We don't fire here any more, is that ok?

Reply:
Yeah, all internal code sets arcStateInternal directly and fires. I'll change this to prevent using the setter (throw error).